app/app.py

```diff
@@ -9,5 +10,36 @@ def hello():
     return 'Hello This is the Flask App'
 
 
+
+""" The following code issues a call to the MET Api, and returns a json object, 
+with the details of 8 pre-determined works of art so that they can be formatted for gameplay.
+ """
+
+@app.route('/api/MetAPI')
```

Comment:
The execution time for this API call averaged around 4.35 seconds for me ... what what the performance for you? This is really slow, I'm wondering if the call to the MET API is slow? I would run some performance measures (time breakpoints) to try and isolate whats slowing you down and can it be adjusted to be better.